browser/brave_content_browser_client.cc

```diff
@@ -572,6 +574,9 @@ void BraveContentBrowserClient::RegisterBrowserInterfaceBindersForFrame(
       brave_private_new_tab::mojom::PageHandler, BravePrivateNewTabUI>(map);
   chrome::internal::RegisterWebUIControllerInterfaceBinder<
       brave_shields::mojom::PanelHandlerFactory, ShieldsPanelUI>(map);
+  chrome::internal::RegisterWebUIControllerInterfaceBinder<
```

Comment:
should be guarded by the feature flag, no?

Reply:
Yes - done.